subworkflows/local/arg.nf

```diff
@@ -21,6 +24,21 @@ workflow ARG {
      // Prepare HAMRONIZATION reporting channel
     ch_input_to_hamronization_summarize = Channel.empty()
 
+    // AMRfinderplus run
+    if ( !params.arg_skip_amrfinderplus ) {
+
+        AMRFINDERPLUS_UPDATE ( )
+        AMRFINDERPLUS_RUN ( contigs, AMRFINDERPLUS_UPDATE.out.db )
+        ch_versions = ch_versions.mix(AMRFINDERPLUS_RUN.out.versions)
+
+    // Reporting
+    // Note: currently hardcoding versions, TODO: has to be updated when amrfinderplus_run is updated to emit
```

Comment:
i thought you added this in the module update?

Reply:
No, not yet, this has to be updated in the AMRFINDER_RUN module and I was on the HAMRONIZATION/AMRFINDER module

Reply:
Ahh sorry, correct!